### DISCUSSION TITLE
Collaborating on PromtFlow pipeline development in ai.azure.com

### DISCUSSION THREAD

**Discussion by HyeonjeCho** [microsoft/promptflow#2374] [Q&A]
I'm working on developing a llm application using PromptFlow on ai.azure.com. 
I'm wondering if there's a way to collaborate on this with other team members. We're all included in the same Azure resource group. 
Is there a way to grant access to my promptflow?

**Comment by brynn-code** [comment]
Sorry for the inconvenience, flow is designed as a personal concept, and there are also some other constraints, for example the runtime, so we can't achieve collaborate flow cross team for now.
